build/package.sh

```diff
@@ -48,8 +47,7 @@ fi
 sed                                \
     -e "s|ARG_BIN|${BIN}|g"        \
     -e "s|ARG_ARCH|${ARCH}|g"      \
-    -e "s|ARG_FROM|${BASEIMAGE}|g" \
     -e "s|ARG_SOURCE_BIN|${SOURCE_BIN}|g" \
     Dockerfile.in > .dockerfile-${ARCH}
-docker build -t ${IMAGE}:${VERSION} -f .dockerfile-${ARCH} .
+docker build ${baseimagearg:-} -t ${IMAGE}:${VERSION} -f .dockerfile-${ARCH} .
```

Comment:
build args make way more sense. I think this pattern was something I copied from Tim Hockins. Let's switch to build args completely in a followup.

Reply:
sounds good to me ;)